doc/src/modules/physics/continuum_mechanics/beam_problems.rst

```diff
@@ -346,36 +346,36 @@ away from start.
     >>> b.reaction_loads
     {M: 157/2, V: -49/2}
     >>> b.load
-           -2         -1                                                                      
-    157⋅<x>     49⋅<x>          0            0          1             -1          0          1
-    ───────── - ──────── + 4⋅<x>  - 4⋅<x - 2>  + <x - 3>  + 12⋅<x - 4>   - <x - 6>  - <x - 6> 
-        2          2                                                                          
+           -2         -1
+    157⋅<x>     49⋅<x>          0            0          1             -1            0          1
+    ───────── - ──────── + 4⋅<x>  - 4⋅<x - 2>  + <x - 3>  + 12⋅<x - 4>   - 6⋅<x - 6>  - <x - 6>
```

Comment:
Just noticed this is slightly incorrect. It should be a 3 and a 6 as the coefficient.